EpiAware/test/test_utilities.jl

```diff
@@ -119,3 +120,51 @@ end
     end
 
 end
+@testitem "Testing neg_MGF function" begin
```

Comment:
no test for the overall approximation?

Reply:
As in the R -> r approximation?

Reply:
yes

Reply:
as in you have it your interactive check but not in an actual test. I think we should just make a new issue for moving these prior and posterior checks into the docs and unit tests tbh